modules/cnf-about-topology-aware-lifecycle-operator-config.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
```

Comment:
I don't think I've seen anything on blocking mechanisms here.

Reply:
@irinamihai I created a new section with basic info and examples. Please check if I missed anything!